### DISCUSSION TITLE
How to Disable Screenshot protection to prepare instructions [Question]

### DISCUSSION THREAD

**Discussion by ademdal** [2fast-team/2fast#45] [Q&A]

**Comment by jp-weber** [comment]
As a security function, this is not possible to deactivate in the Store version in a simple way. 

If the app is started in debug mode, the protection setting is deactivated by default. As an option, I recommend installing Visual Studio 2022 and compiling the application with the source code.

**Comment by jp-weber** [comment]
@ademdal 
Would a debug version help, with the anti screenshot function and other functions deactivated? Of course, it should be borne in mind that this is not suitable for productive use and loads significantly more slowly.

**Comment by ademdal** [comment]
Thank you very much @jp-weber for your advice. 
As I'm not using Visual Studio for 10 years, I will get help form a friend this week. I just need it to create instructions on how to add secrets. 
I'm planning to use Project2FA.UWP_1.2.0.0_Store_Installer to package and deploy.
